City-of-Helsinki/helsinki-paatokset
# PP-414 & PP-418: Decision history migrations, decision history HTML parsing, image data URL fixes

**To test**
- Checkout branch, run `make drush-cim drush-cr`
- SSH into the container with `make shell` and run the following commands to import test data:
```
drush ap:update decisions {7FD6BD51-71BD-46D4-8C0B-7B737AAEB075} -v
drush ap:update decisions {B37F73CD-E527-4403-BDDB-ECF78CA6C32E} -v
drush ap:update decisions {1E008A20-9926-45CF-BD8D-E7289C138422} -v
drush ap:update decisions {FA858AE5-E02F-4C75-A16A-FB4ADCE61ADE} -v
drush ap:update decisions {5E656BD7-6CA8-CB8B-BB67-832121900005} -v
drush ap:update decisions {1C234593-D74E-44AB-9238-411059D72559} -v
drush ap:update decisions {3718C754-2FFF-49DF-BC12-1EEB48686C47} -v
drush ap:update decisions {BD026030-8442-44BA-851A-15FEFB25AE49} -v
drush ap:ud -v
drush queue:run ahjo_api_subscriber_queue -v
drush ap:ud --update -v
```
- Open the following decisions and check that they have the "Decision history" accordion and that it has content:
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2021-008928/1e008a20-9926-45cf-bd8d-e7289c138422
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2022-008455/7fd6bd51-71bd-46d4-8c0b-7b737aaeb075
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2022-006842/b37f73cd-e527-4403-bddb-ecf78ca6c32e
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2022-007652/fa858ae5-e02f-4c75-a16a-fb4adce61ade 
  - The first three should have simple content inside the accordion, but the last one has headings, subheadings and text
- Open the following decisions and check that they have one or more images visible on the page and that there are no fatal errors. The image should actually load (not just a broken image icon):
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2022-003331/5e656bd7-6ca8-cb8b-bb67-832121900005 (image is table with text)
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2019-001785/1c234593-d74e-44ab-9238-411059d72559
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2018-004863/3718c754-2fff-49df-bc12-1eeb48686c47
  - https://helsinki-paatokset.docker.so/fi/asia/hel-2021-009847/bd026030-8442-44ba-851a-15fefb25ae49
